interpreter/cling/include/cling/Interpreter/CompilationOptions.h

```diff
@@ -73,6 +78,7 @@ namespace cling {
 
     CompilationOptions() {
       DeclarationExtraction = 0;
+      EnableShadowing = 0;
```

Comment:
How / where is `EnableShadowing` enabled?

Reply:
We talked about that this morning; (but for completeness) I will reply. :-)
See lib/Interpreter.cpp:782 (enable if not using raw input).

Reply:
So in the end we will enable shadowing by default when you start cling and not using raw input?